### PR TITLE
Run kindtest on GCE instance

### PR DESCRIPTION
**Pull request by mitsutaka**
It allows developer to debug on the instance if CI is failed.

### CODE REVIEW

File: bin/run-kindtest.sh [hunk 1]
```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
+
+. ./bin/env
+
+# Create GCE instance
+$GCLOUD compute instances delete ${INSTANCE_NAME} --zone ${ZONE} --quiet || true
+$GCLOUD compute instances create ${INSTANCE_NAME} \
+  --zone ${ZONE} \
+  --machine-type ${MACHINE_TYPE} \
```
Comment (zoetrope) [inline bin/run-kindtest.sh]: I think it can be a small instance.
How do you think?

File: bin/run-kindtest.sh [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+# Overwrite daemon.json for using default docker0 IP address
+cat >/etc/docker/daemon.json << EOF2
+{
+    "bip": "172.17.0.1/16"
+}
+EOF2
+sudo systemctl restart docker.service
+
+cd test
+cp /home/cybozu/account.json ./
```
Comment (zoetrope) [inline bin/run-kindtest.sh]: Is `account.json` necessary?
Currently, kindtest does not access CloudDNS.

File: bin/run-kindtest.sh (hunk 1 above)
Comment (mitsutaka) [reply bin/run-kindtest.sh]: Let me try on similar machine-type
https://circleci.com/docs/2.0/executor-types/#using-machine
2	Intel(R) Xeon(R) @ 2.3GHz	8GB	100GB

Comment (mitsutaka) [reply bin/run-kindtest.sh]: https://circleci.com/gh/cybozu-go/neco-apps/4045 29mins

n1-standard-2

```console
 00:01:25 up 26 min,  1 user,  load average: 18.17, 18.16, 12.36

              total        used        free      shared  buff/cache   available
Mem:        7652348     4536040      142912       38248     2973396     2970712
Swap:             0           0           0
```

Comment (mitsutaka) [reply bin/run-kindtest.sh]: n1-standard-4 https://circleci.com/gh/cybozu-go/neco-apps/4047 21:47

Comment (mitsutaka) [reply bin/run-kindtest.sh]: n1-standard-8 https://circleci.com/gh/cybozu-go/neco-apps/4050 21:40

```console
00:46:45 up 18 min,  1 user,  load average: 3.43, 6.42, 4.86

              total        used        free      shared  buff/cache   available
Mem:       30875560     5000092     8659176       38280    17216292    25588872
Swap:             0           0           0
```

n1-standard-4 seems enough.

Comment (zoetrope) [reply bin/run-kindtest.sh]: :+1:

**Pull request merged**